src/modules/post.js

```diff
@@ -0,0 +1,18 @@
+const postValue = async (formData, gameAPI) => {
+  await fetch(gameAPI, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(
+      formData,
+    ),
+  })
+    .then((response) => response.json())
+    .then((data) => data.result);
+//     .then((response) => response.json())
+//     .then((data) => console.log(data));
+//   // .then((data) => data.result);
```

Comment:
- [ ] Kindly make sure you delete the commented code as they make your code base looks ugly.